cmsl1t/hist/factory.py

```diff
@@ -1,6 +1,7 @@
 from cmsl1t.hist import BaseHistogram
 import logging
 import rootpy.plotting.hist as rootpy_hists
+import rootpy.ROOT as ROOT
```

Comment:
For building the documentation it is useful to import ROOT dependencies late (e.g. for readthedocs).
Could you please move them into the relevant functions?
Alternatively, we need to add conda support: http://docs.readthedocs.io/en/latest/conda.html.
I think I would prefer the latter in the long run.